docs/graphman-graphql-api.md

```diff
@@ -0,0 +1,212 @@
+# Graphman GraphQL API
+
+When starting the `graph-node` optionally, a graphman GraphQL server can be started, which makes it possible to execute
+graphman commands via GraphQL.
+
+To start the graphman GraphQL server, the `GRAPHMAN_SERVER_AUTH_TOKEN` environment variable should be set. The token is
+used to authenticate graphman GraphQL requests.
+
+Other environment variables that can be set:
```

Comment:
I would just say `Environment variables to control the graphman API` here and also list the var for the token (again) That makes it easier to see all of them for reference

Reply:
Updated with the proposed version